src/Controller/QueueTrigger.php

```diff
@@ -24,14 +25,17 @@ public function processToEndTime() {
       throw new NotFoundHttpException();
     }
     else {
-      // The endpoint has been called successfully.
-      // @Todo Call the queue runner service.
-      // Log it.
-      $vars = ['@time' =>  date('m-d-Y H:i:s', $time)];
+      // Process our bulletins.
+      $trigger = new BulletinBase();
+      $status = $trigger->processQueue($time);
+
+      // The endpoint has been called.
+      // Log status.
+      $vars = ['@time' => date('m-d-Y H:i:s', $time)];
       $build = [
-        '#markup' => $this->t('The queue has been triggered for anything prior to: @time', $vars),
+        '#markup' => $this->t('The queue has been triggered for anything prior to: @time and returned this status: @status', [$vars, $status]),
```

Comment:
```suggestion
        '#markup' => $this->t('The queue has been triggered for anything prior to: @time and returned this status: @status', $vars),
```
I don't think that would work to have the other vars nested one level deeper.

Reply:
:+1: